src/service/ui/component/Page.html

```diff
@@ -20,9 +20,11 @@
     <Header />
 
     <slot/>
+    <script src="./js/govuk_frontend_toolkit/govuk/show-hide-content.js"></script>
+    <script src="./js/govuk_frontend_toolkit/govuk/modules.js"></script>
     <script src="./js/main.js"></script>
-        <script src="./js/govuk_frontend_toolkit/govuk/show-hide-content.js"></script>
-        <script src="./js/govuk_frontend_toolkit/govuk/modules.js"></script>
+    <script>if (typeof window.GOVUK === 'undefined') document.body.className = document.body.className.replace('js-enabled', '');</script>
```

Comment:
Does this work? Doesn't svelte complain about this?

Reply:
Yes it works, and no svelte doesn't complain, maybe because it's the parent? unsure 😅

Reply:
Yay!